sqlstore/src/main/resources/db/migration/V0320__auto_main.sql

```diff
@@ -45,11 +44,11 @@ CREATE TABLE `PoolQC2` (
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
 INSERT INTO LibraryQC2(`qcId`, `library_libraryId`, `creator`, `date`, `type`, `results`)
-	SELECT qcId, library_libraryId, (SELECT userId FROM `User` WHERE fullName = qcCreator), qcDate, qcMethod, results FROM LibraryQC;
+	SELECT qcId, library_libraryId, IFNULL((SELECT userId FROM `User` WHERE fullName = qcCreator OR loginName = qcCreator),1), qcDate, qcMethod, results FROM LibraryQC;
```

Comment:
1. I don't think a substitute user is appropriate. If your database has bad data, it should fail to migrate.
2. Why user 1? It's not even guaranteed to be admin. I believe it isn't admin in your database.

Reply:
I'm unsure how to represent this in this instance. We can't delete the QCs. We could map them to an existing user in our DB, but then we'll lose the real data. The only solution we have if we can't map it to an existing user is to create a new user for each external QC creator. I'll speak to the team here.